components/eventing-controller/pkg/handlers/metrics/collector.go

```diff
@@ -59,18 +56,18 @@ func (c *Collector) Collect(ch chan<- prometheus.Metric) {
 	c.eventTypes.Collect(ch)
 }
 
-// RecordDeliveryPerSubscription records the delivery_per_subscription metric
-func (c *Collector) RecordDeliveryPerSubscription(subscriptionName, eventType, sink string, statusCode int) {
-	c.deliveryPerSubscription.WithLabelValues(subscriptionName, eventType, fmt.Sprintf("%v", sink), fmt.Sprintf("%v", statusCode)).Inc()
-}
-
 // RegisterMetrics registers the metrics
 func (c *Collector) RegisterMetrics() {
 	metrics.Registry.MustRegister(c.deliveryPerSubscription)
 	metrics.Registry.MustRegister(c.eventTypes)
 }
 
-// RecordEventTypes records the event_type_subscribed metric
+// RecordDeliveryPerSubscription records a eventing_ec_delivery_per_subscription_total metric
+func (c *Collector) RecordDeliveryPerSubscription(subscriptionName, eventType, sink string, statusCode int) {
+	c.deliveryPerSubscription.WithLabelValues(subscriptionName, eventType, fmt.Sprintf("%v", sink), fmt.Sprintf("%v", statusCode)).Inc()
+}
+
+// RecordEventTypes records the jetstream_ec_event_type_subscribed_total metric
```

Comment:
(nit) any reason for using "a"/"an"/"the" in different comments?

Reply:
I tried to be consistent with using a/an but apparently I missed a few. Will go over it again!